airflow/providers/common/sql/hooks/sql.py

```diff
@@ -294,6 +297,22 @@ def _run_command(self, cur, sql_statement, parameters):
         if cur.rowcount >= 0:
             self.log.info("Rows affected: %s", cur.rowcount)
 
+    def _update_query_ids(self, cursor) -> None:
+        """
+        Adds query ids to list
+        :param cur: current cursor after run
+        :return:
+        """
+        return None
+
+    def stop_query(self, query_id) -> Any:
```

Comment:
there is some confusing terminology. if we use the term `kill` lets stick with it?

Reply:
I just saw that for Athena and EMR there's a similar method `stop_query` so I decided to use that name

Reply:
I don't really mind about specific name just about consistency/standartization.
There are places in the projects where names were just given without too much thought so I'm raising it whenever I can and whenever appropriate.
It's OK also to decide to keep as is.

In any case - naming while important are not the priority. We can leave that point after functionality works as expected.